lib/compilers/clspv.ts

```diff
@@ -41,7 +41,7 @@ export class CLSPVCompiler extends BaseCompiler {
     constructor(compilerInfo, env) {
         super(compilerInfo, env);
 
-        this.asm = new SPIRVAsmParser();
+        this.asm = new SPIRVAsmParser(this.compilerProps);
```

Comment:
Is this related to the cc65 or maybe it's a fix we need to make on main separately?

Reply:
No was a type/inheritance bugfix